src/lib/versionOpt.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
Could add this to the base `Command` instead and then do `this.getVersionOption()` in `this.args` instantiations. We could do the same for the common `key` arg that most every command has too.

Reply:
Yeah I thought about this but wasn't sure how to do so elegantly—some (but not all) commands use this opt, and then in some cases it uses a `version` opt with a different description:
https://github.com/readmeio/rdme/blob/2cd9847ad6c305f7e6db2e26148852d655ff9720/src/cmds/versions/index.ts#L43-L47

Reply:
How about `this.getVersionOption('A specific project version to view')`> And then in the `getVersionOption` you can have a `description` type that's:

```ts
description: 'A specific project version to view' | 'the other description we use'
```

Could also make those descriptions an enum and then do `VersionOptionDescription.SPECIFIC` and `VersionOptionDescription.GENERAL`.